ChartboostMediationAdapterVungle.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |spec|
   spec.module_name  = 'ChartboostMediationAdapterVungle'
   spec.source       = { :git => 'https://github.com/ChartBoost/chartboost-mediation-ios-adapter-vungle.git', :tag => spec.version }
   spec.source_files = 'Source/**/*.{swift}'
+  spec.resource_bundles = { 'ChartboostMediationAdapterVungle' => ['Source/PrivacyInfo.xcprivacy'] }
```

Comment:
again, not a source file. can we put it in the parent directory?

Reply:
updated now